third_party/nvfuser/csrc/index_compute.cpp

```diff
@@ -563,8 +563,38 @@ void IndexCompute::handle(Swizzle2D* swizzle_2d) {
   }
 }
 
+void IndexCompute::handle(Resize* resize) {
+  auto out_id = maybeGetExactMapConcreteID(resize->out());
+  auto in_id = maybeGetExactMapConcreteID(resize->in());
+
+  auto out_it = index_map_.find(out_id);
+
+  if (out_it == index_map_.end()) {
+    return;
+  }
+
+  const auto out_ind = out_it->second;
+
+  auto zero = GpuLower::current()->kernel()->zeroVal();
+
+  if (isZero(out_id)) {
+    index_map_[in_id] = zero;
```

Comment:
Actually, when this happens, indexing the input ID is invalid. Being zero effectively means that all of the producer and consumer tensors access the domain with the same index, and there should be no such case with resize, at least as this point. For a producer tensor, since inlining is not allowed with resized domains, out_id should never be a zero domain. For a consumer tensor, resize should only appear as part of the rfactor transformations, and since indexing is done with the rfactor root domain, resize should never be involved.

Updated: 622ee939a5038f4d86dcf496cf371e313073051c